contrib/dokku-installer.py

```diff
@@ -95,6 +95,9 @@ def do_POST(self):
                 os.remove('{0}/VHOST'.format(dokku_root))
             except OSError:
                 pass
+        with open('{0}/HOSTNAME'.format(dokku_root), 'w') as f:
```

Comment:
Shouldn't it be outside of the `if ... else`? I think `HOSTNAME` should get written regardless of whether or not virtualhost naming is used, no?